src/definitions/elements/loader.less

```diff
@@ -175,6 +175,68 @@
   font-style: normal;
 }
 
+/*-------------------
+        Dots
+--------------------*/
+.ui.dots.dots.dots.loader,
+.ui.dots.dots.dots.loader::after,
+.ui.dots.dots.dots.loader::before {
+  border-width: 0;
+  width: @dotsDiameter;
+  height: @dotsDiameter;
+  border-radius: @dotsRadius;
+  animation-name: defaulDotsLoader;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
```

Comment:
Refactored in https://github.com/fomantic/Fomantic-UI/pull/723/commits/a10dee43021f5385016987f4ca6ca4f5e6efce7b

Reply:
Sorry, i mistyped it. Of course we need `background-color: currentColor` instead of `border-color: currentColor` 🙄  because the dots are using the background. The border is invisible anyway by setting the width to 0... (i wonder if `border:none;` is probably better?)